server.js

```diff
@@ -12,6 +12,16 @@ if (process.env.TEST_ENV) {
     return;
 }
 
+console.log('Node Version:', process.version);
+var version = process.versions.node.match(/(\d)+\.(\d)+\.(\d)+/)[1];
+if (Number(version[1] <= 4)) {
+  // 5 seems to be where the majority of ES6 was added without flags.
+  // Seems safeist to not allow api to run
+  throw new Error('Node version ' + process.version + ' is not supported');
+}
+if (Number(version[1]) < 6 && Number(version[2]) < 10) {
+  console.error('This node version may not be supported');
```

Comment:
Hmm I think I'm losing my mind: I swear `console.warn` didn't used to exist… it seems to exist in both node and the browser though, so IDK. Must be thinking of something else. I'll change it.

In terms of changing the version, maybe we should line up travis and medic-os then? The wording doesn't come from what medic-os ships, it comes from what we officially test on. So either: we should change travis or we should change medic-os. Thoughts @browndav / @garethbowen ?